roles/nat64_appliance/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure needed directories exist
```

Comment:
@cjeanner - nothing is now captured by CI ... Not quite what we want.

I am placing `artifacts` and `logs` under subdirectory `ci-framework-data/nat64_appliance`, but nothing is capturing that. Should I change the `cifmw.general.ci_script` to - `{{ cifmw_nat64_appliance_basedir }}/artifacts`? Or is something supposed to pick up all subdirectories names `artifacts` and `logs` ?

It is not in the molecule job - https://logserver.rdoproject.org/36/1536/975011ab03234110d1e3845e1c7e2207bf3a3fc5/github-check/cifmw-molecule-nat64_appliance/8e0d3a2/

Reply:
I changed the paths, we should capture files under artifacts/ and logs/ now.

Marking resolved.